crates/cxx-qt-gen/test_outputs/passthrough_and_naming.cpp

```diff
@@ -279,7 +279,7 @@ MyObject_readyConnect(
 
 namespace cxx_qt::multi_object {
 MyObject::MyObject(QObject* parent)
-  : QStringListModel(parent)
+  : cxx_qt::multi_object::QStringListModel(parent)
```

Comment:
this is not quite right, but i think it's because the input has a namespace on the bridge?

Can you add a `#[namespace = ""]` onto the `QStringListModel` or it's extern block so we have a sane value here?